.github/workflows/deploy-ghpages.yml

```
@@ -0,0 +1,34 @@
+name: Deploy to GitHub pages
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+  release:
+    types:
+      - published
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "22.16.0"
+
+      - name: Install and Build
+        run: |
+          cd documentation
+          npm install
```

```suggestion
          npm ci
```
I think this is safer since it will be a known set of dependencies from the `package.lock` whereas `npm install` will get the latest versions that `package.json` allows.